README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
> I'd argue that enhancing tests for older versions of ECMAScript is relatively low priority compared to expanding coverage for the ToT spec, however.

I would be very surprised if anyone disagreed with this ;)

> To properly test old versions, we would really need version ceilings, not just version floors.

Yeah, and I think this is fine. Previously valid tests made invalid are deleted (or updated to have the new versions' metadata attribute). Otherwise, running tests that only have an esNid where N <= the version you want to test should be a very close approximation.